code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -71,6 +72,7 @@
 
 /obj/item/reagent_containers/blood/ethereal
 	blood_type = "LE"
+	unique_blood = "liquidelectricity"
```

Comment:
because making the r coloegeantr per race color is effort, I've decided that the ethereal skin defines their glow color but their blood is always yellow-ish.

Reply:
kinda like when u put a light under a red filter the light seems to be red